apps/booking-app/pages/index.js

```diff
@@ -12,11 +12,13 @@ import {
   LoginButton,
   UserAccount,
   FavoriteCard,
-  Booking
+  Booking,
+  ResultsPage
 } from '@booking-ui/shared';
 
```

Comment:
You shouldn’t include the result's page here, to view this page you must enter the route */results*